Blockzilla/OverlayView.swift

```diff
@@ -198,3 +202,12 @@ extension OverlayView: KeyboardHelperDelegate {
     func keyboardHelper(_ keyboardHelper: KeyboardHelper, keyboardDidShowWithState state: KeyboardState) {}
     func keyboardHelper(_ keyboardHelper: KeyboardHelper, keyboardDidHideWithState state: KeyboardState) {}
 }
+
+extension UIPasteboard {
```

Comment:
I think this is fine here for now. But I think we should start looking for a more permanent place for these utility extensions. We're starting to add a few of them